src/harness/reference_models/interference/aggregate_interference.py

```diff
@@ -35,11 +35,16 @@
 """
 
 import numpy as np
+from functools import partial
+
+from reference_models.common import mpool
+from reference_models.common import data
 from reference_models.interference import interference as interf
 from reference_models.geo import utils
-import multiprocessing
-from multiprocessing import Pool
-from functools import partial
+
+# The grid resolution for area based protection entities.
+GWPZ_GRID_RES_ARCSEC = 10
+PPA_GRID_RES_ARCSEC = 10
```

Comment:
Shouldn't this be 2 arc sec for area protection?

Reply:
Yes sorry, I was temporarily changing them to run regression test which do not last too long. Just fixed. this